.github/workflows/dotnet-ci.yml

```
@@ -0,0 +1,54 @@
+            /d:sonar.cs.opencover.reportsPaths="${{ github.workspace }}/coverage.xml" `
+            /d:sonar.coverage.exclusions="**/Program.cs" `
+            /d:sonar.sourceEncoding="UTF-8" `
+            /d:sonar.language="cs"
+
+      - name: Build solution
+        run: dotnet build --no-restore
+
+      - name: End SonarCloud analysis
+        env:
+          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+        run: dotnet sonarscanner end /d:sonar.token="${{ secrets.SONAR_TOKEN }}"
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)